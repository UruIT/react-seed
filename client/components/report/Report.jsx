import React from 'react';
import style from './Report.scss';

const Report = () =>
	<form action="api/report" className={style.report}>
		<h3> Click below to generate a pdf report using wkhtmltopdf </h3>
		<input type="submit" value="Download pdf" />
	</form>

export default Report;
