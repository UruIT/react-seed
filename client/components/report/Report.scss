.report {
	text-align: center;

	input[type="submit"] {
		display: block;
		padding: 10px 20px;
		border-radius: 5px;
		margin: 10px auto;
	}
}
