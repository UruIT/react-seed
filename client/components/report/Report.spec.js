import React from 'react';
import Report from './Report';
import renderer from 'react-test-renderer';

test('renders Report', () => {
	const tree = renderer.create(<Report />).toJSON();

	expect(tree).toMatchSnapshot();
});
