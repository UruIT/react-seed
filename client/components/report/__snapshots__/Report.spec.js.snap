// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Report 1`] = `
<form
  action="api/report"
  className="report"
>
  <h3>
     Click below to generate a pdf report using docraptor 
  </h3>
  <input
    type="submit"
    value="Download pdf"
  />
</form>
`;
